_includes/footer.html

```diff
@@ -1,12 +1,21 @@
 <footer class="ds-base ds-u-fill--white ds-u-margin-top--7 ds-u-padding-y--6">
   <div class="ds-l-container">
     <div class="ds-l-row">
-      <div class="ds-u-font-size--small ds-l-col--12 ds-l-sm-col--6 ds-l-md-col--6 ds-u-margin-bottom--2">
+      <div class="ds-u-font-size--small ds-l-col--12 ds-l-sm-col--6 ds-l-md-col--3 ds-u-margin-bottom--2">
         <div class="ds-u-float--left ds-u-margin-right--2">
           {% include HHS-logo.svg %}
         </div>
         A federal government website managed by the Centers for Medicare &amp; Medicaid Services 7500 Security Boulevard, Baltimore, MD 21124
       </div>
+      <div class="ds-l-col--12 ds-l-sm-col--6 ds-l-md-col--3 ds-u-margin-bottom--2">
+        <h2 class="ds-h4">Beneficiary Claims Data API</h2>
+        <ul class="ds-c-list ds-c-list--bare ds-u-font-size--small">
+          <li><a target="_blank" href="https://bcda.cms.gov/#overview">Documentation</a></li>
```

Comment:
This should not open in a new tab, since its a link to an internal page.  Remove the `target="_blank"`